.circleci/config.yml

```diff
@@ -214,23 +201,42 @@ jobs:
           name: Check the spec tests for idempotence
           command: postgrest-test-spec-idempotence
           when: always
+      - run:
+          name: Run linter
+          command: postgrest-lint
+          when: always
+      - run:
+          name: Run style check
+          command: postgrest-style-check
+          when: always
```

Comment:
Can we run those as early as possible? Always a pain to have everything pass but some whitespace is wrong at the end.

Reply:
I suggest you add a pre-push hook to your git repo running at least style-check - would that be early enough? :D

I'll put them first after the build.

Reply:
> I suggest you add a pre-push hook to your git repo running at least style-check - would that be early enough? :D

Pre-commit is even earlier!

Reply:
Snarky! But you're right.. Could we get a `postgrest-install-git-hooks` that sets up the recommended Wolfgang config?